src/core/cli/lifecycle/layout_test.go

```
@@ -0,0 +1,96 @@
+import (
+	"os"
+	"path/filepath"
+	"strings"
+	"testing"
+)
+
+func TestRootDefault(t *testing.T) {
+	// Clear any existing override and env var.
+	defer WithRoot("")()
+	prev := os.Getenv(envHome)
+	os.Unsetenv(envHome)
```

golangci-lint (2.11.4)

[error] Error return value of `os.Unsetenv` is not checked

(errcheck)